acceptance/bundle/variables/arg-repeat/script

```diff
@@ -0,0 +1,2 @@
+trace $CLI bundle validate --var a=one -o json | jq .variables
+trace errcode $CLI bundle validate --var a=one --var a=two
```

Comment:
No output for this one?

Reply:
Ah, it should be reversed; "errcode trace".

Reply:
Good catch. I missed errcode - added now.

Reply:
"trace errcode" is proper sequence.